scripts/v2maker.js

```diff
@@ -408,15 +419,16 @@ async function makeTestsCsvData(allInputCsvData, assertionRows) {
     }
   }
 
-  // Now filter v2testRows down to just the VoiceOver rows.
-  // First, get an array of just the voiceOver rows from v1tests.
-  const v1voiceOverRows = allInputCsvData.v1tests.filter(row =>
-    row.appliesTo.toLowerCase().includes('voiceover')
-  );
-  // Build a set  of the v1 testIds for the VoiceOver tests.
-  const voiceOverV1testIds = new Set(v1voiceOverRows.map(row => row.testId));
-  // Filter v2testRows. Note that the presentationNumber property contains the v1testId.
-  v2testRows = v2testRows.filter(row => voiceOverV1testIds.has(row.presentationNumber));
+  const v2uniqueTestRows = [];
+  const seenTestIds = new Set();
+
+  for (const row of v2testRows) {
+    if (!seenTestIds.has(row.testId)) {
+      v2uniqueTestRows.push(row);
+      seenTestIds.add(row.testId);
+    }
+  }
+  v2testRows = v2uniqueTestRows;
```

Comment:
The existence of a VoiceOver test row was being used to assume that the equivalent JAWS + NVDA to properly populate `v2testRows` but that wasn't always the case.

Reply:
This block has been updated as it was leaking in instances of `reading` and `interaction` in the titles of the outputted files but the reasoning remains the same.